core/domain/email_manager.py

```diff
@@ -350,6 +383,56 @@ def _send_email_in_transaction():
     transaction_services.run_in_transaction(_send_email_in_transaction)
 
 
+def _send_emails(send_email_infos):
```

Comment:
Could we de-duplicate this with ``_send_email``? There seems to be a lot of overlap.

Reply:
Which check in particular? I think we should try to de-duplicate if possible given the amount of overlap.